extensions/wikia/AdEngine/js/video/player/ui/toggle.js

```diff
@@ -0,0 +1,33 @@
+/*global define, setTimeout*/
+define('ext.wikia.adEngine.video.player.ui.toggle', [
+	'ext.wikia.adEngine.domElementTweaker'
+], function (DOMElementTweaker) {
+	'use strict';
+
+	function toggle(elementToShow, elementToHide) {
+		DOMElementTweaker.hide(elementToHide);
+		DOMElementTweaker.show(elementToShow);
+	}
+
+	function hideVideo(video, params) {
+		toggle(params.videoPlaceholderElement || params.image, video.container);
```

Comment:
we can avoid `params.videoPlaceholderElement || params.image` by doing this or statement when passing params to `videoInterface.setup` - https://github.com/Wikia/app/pull/12250/files#diff-88f79fd88ef24f1113a283bf78cfaf7eR46